client/src/components/Pages/CausativeEvent/CausativeEvent.tsx

```diff
@@ -9,6 +9,7 @@ import {
   RadioGroup,
   TextField,
 } from "@material-ui/core";
+import React from "react";
```

Comment:
This is not needed, see [here](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform).  Should remove all places where you added these imports. 
```suggestion
```